mbq/api_tools/views.py

```diff
@@ -36,7 +36,7 @@ def __init__(
         on_unknown_field: Optional[OnUnknownField] = None,
         _method: Optional[bool] = None,
     ):
-        if not permissions and settings.API_TOOLS.get("REQUIRE_PERMISSIONS", True):
+        if not permissions:
```

Comment:
The `kwarg` above should remove the `Optional`: `permissions: PermissionsCollection = ()`